numcosmo/lss/nc_galaxy_wl.c

```diff
@@ -289,13 +296,83 @@ nc_galaxy_wl_eval_m2lnP (NcGalaxyWL *gwl, NcHICosmo *cosmo, NcHaloDensityProfile
   NcGalaxyWLEval gwleval         = {self->wl_dist, cosmo, dp, smd, z_cluster, 0};
   gdouble res                    = 0.0;
   gint gal_i;
+
+  nc_galaxy_wl_dist_m2lnP_initial_prep (self->wl_dist, self->gz_dist, cosmo, dp, smd, z_cluster);
   
+  // if (self->no_kde)
```

Comment:
Remove all old code in comments. This is not necessary as we can find all this in the previous commit.